data/blog/announcing-mirage-40.md

```diff
@@ -0,0 +1,179 @@
+---
+updated: 2022-03-27
+authors:
+- name: Thomas Gazagnaire
+  uri: https://gazagnaire.org
+subject: Announcing MirageOS 4.0.0
+permalink: announcing-mirage-40
+---
+
+**On behalf of the Mirage team, I am delighted to announce the release
+  of MirageOS 4.00!**
+
+
+Since the first release of 2013, MirageOS has made steady progress
+towards deploying a self-managed internet infrastructure. The
+project’s initial aim was to self-host as many services as possible
+aimed at empowering internet users to securely deploy infrastructure
+to own their data and take back control of their privacy. MirageOS can
+securely deploy [static website
+hosting](https://github.com/roburio/unipi) with “Let’s Encrypt”
+certificate provisioning and a [secure SMTP
+stack](https://github.com/mirage/ptt) with security
+extensions. MirageOS can also deploy decentralised communication
+infrastructure like [Matrix](https://github.com/mirage/ocaml-matrix),
+[OpenVPN servers](https://github.com/roburio/openvpn), and [TLS
+tunnels](https://github.com/roburio/tlstunnel) to ensure data privacy
+or [DNS(SEC) servers](https://github.com/mirage/ocaml-dns) for better
+authentication.
+
+The protocol ecosystem now contains hundreds of libraries and services
+millions of daily users. Over these years, major commercial users have
+joined the projects. They rely on MirageOS libraries to keep their
+products secure. For instance, the MirageOS networking code powers
+[Docker Desktop’s
+VPNKit](https://www.docker.com/blog/how-docker-desktop-networking-works-under-the-hood/),
+which serves the traffic of millions of containers daily. [Citrix
+Hypervisor](https://www.citrix.com/fr-fr/products/citrix-hypervisor/)
+uses MirageOS to interact with Xen, the hypervisor that powers most of
+today’s public
+cloud. [Nitrokey](https://www.nitrokey.com/products/nethsm) is
+developing a new hardware security module based on
+MirageOS. [Robur](https://robur.io/) develops a unikernel
+orchestration system for fleets of MirageOS
+unikernels. [Tarides](https://tarides.com/) uses MirageOS to improve
+the [Tezos](https://tezos.com/) blockchain, and
+[Hyper](https://hyper.ag/) uses MirageOS to build sensor analytics and
+an automation platform for sustainable agriculture.
+
+In the coming weeks, our blog will feature in-depth technical contents
+for the new features that MirageOS brings, as well as a tour of
+existing community and commercial users of MirageOS. Please reach out
+If you’d like to tell us about your story.
+
+## Install MirageOS 4.00
+
+The easiest way to install MirageOS 4.00 is by using the [Opam package
+manager](https://opam.ocaml.org/) version 2.1. Follow the
+[installation guide](https://mirage.io/docs/install) for more details.
+
+```
+$ opam update
+$ opam install mirage.4.0.0
```

Comment:
```suggestion
$ opam install mirage
```

Best not to have a version number here... or it'll not work with point releases in the future?

Reply:
I'm just afraid that people will continue to have mirage 3 installed without noticing it.